### PR TITLE
Wedding

### PR DESCRIPTION
**Pull request by cristianetanaka**
Hi Sandra,

Please review. Kindly note that this is currently using dummy data. To be tested again when Frontend is connected to Backend.

### CODE REVIEW

File: src/components/WeddingSection.jsx
Comment (Sandz-Studio) [inline src/components/WeddingSection.jsx:1]: This whole file is not needed. There is not wedding section on the home page. Please delete.

File: src/main.jsx
Comment (Sandz-Studio) [inline src/main.jsx:1]: yep good

File: src/pages/WeddingPage.jsx
Comment (Sandz-Studio) [inline src/pages/WeddingPage.jsx:1]: yep good. 
image

The text needs to be white in line with all the other banners. If you use a pink coloured background picture the background behind the heading needs to be blue. Atm the combination you have is not colour accessible

File: src/components/NavBar.jsx
Comment (Sandz-Studio) [inline src/components/NavBar.jsx:1]: yep good

**Review by Sandz-Studio — CHANGES_REQUESTED** [review]

File: src/pages/WeddingPage.jsx
Comment (cristianetanaka) [reply src/pages/WeddingPage.jsx:1]: Changed to Blue picture background. But white colour text is not accessible with pink
image (2)
We can change this later

Comment (Sandz-Studio) [reply src/pages/WeddingPage.jsx:1]: all good i have chnaged the pink! :)

**Review by Sandz-Studio — APPROVED** [review]

**Review by Sandz-Studio — APPROVED** [review]

**Review by Sandz-Studio — APPROVED** [review]

**Pull request merged**